src/main/java/core/basesyntax/serviceimpl/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> readFromFile(String fileName) {
+        try {
+            return Files.readAllLines(Path.of(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t find the file " + fileName, e);
+        }
+    }

```

Comment:
According to the checklist, you should avoid using absolute paths and prefer relative ones. Make sure that the `fileName` parameter is a relative path and consider validating it before using it.